code/modules/mob/dead/observer/observer.dm

```diff
@@ -146,8 +146,12 @@ GLOBAL_VAR_INIT(observer_default_invisibility, INVISIBILITY_OBSERVER)
 	grant_all_languages()
 	show_data_huds()
 	data_huds_on = 1
-
 	SSpoints_of_interest.make_point_of_interest(src)
+
+	/// Ambiant sound prefrence check. Why does this exist here? Because ambience relies on Area code. Ghosts are outside area code and this is the least expensive work around then to add undeeded area checks to observers.
+	if(!(client?.prefs.read_preference(/datum/preference/toggle/sound_ship_ambience)))
+		SEND_SOUND(src, sound(null, repeat = 0, wait = 0, channel = CHANNEL_BUZZ))
```

Comment:
I feel like `SEND_SOUND(src, sound(null, repeat = 0, wait = 0, channel = CHANNEL_BUZZ))` should be in `/datum/preference/toggle/sound_ship_ambience/apply_to_client_updated()`.

So no matter what mob you are or where you are or anything, if you toggle the sound off, it turns it off.